package.json

```diff
@@ -103,8 +103,10 @@
           },
           "snyk.advanced.customEndpoint": {
             "type": "string",
-            "markdownDescription": "Sets API endpoint to use for Snyk requests. Useful for custom Snyk setups. E.g. `https://app.eu.snyk.io/api`.",
-            "scope": "window"
+            "markdownDescription": "Sets API endpoint to use for Snyk requests. Useful for custom Snyk setups. E.g. `https://api.eu.snyk.io`.",
+            "scope": "window",
+            "format": "uri",
+            "pattern": "^(https?://)api.?[a-zA-Z0-9]{0,19}.(snyk|snykgov).io$"
```

Comment:
The only thing to call out here is that I noticed in IntelliJ the custom endpoint supports `http(s)://snyk.io/...` endpoints. VSCode never did though and I'm not sure why